notebooks/224-3D-segmentation/224-3D-segmentation.ipynb

```diff
@@ -0,0 +1,240 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3D part segmentation with OpenVINO\n",
+    "\n",
+    "This notebook demonstrates how to process the point cloud data and run 3D part segmentation with OpenVINO. We use the [PointNet](https://arxiv.org/abs/1612.00593) pre-trained model to detect each part of chair and return its category.\n",
+    "## PointNet\n",
+    "PointNet was proposed by a researcher at Stanford University in 2016. The motivation behind this paper is to classify and segment 3D representation of images. They use a data structure called Point cloud, which is a set of the point that represents a 3D shape or an object. PointNet provides a unified architecture for applications ranging from object classification, part segmentation, to scene semantic parsing. It is highly efficient and effective, showing strong performance on par or even better than state of the art. \n"
```

Comment:
```suggestion
    "PointNet was proposed by a researcher at Stanford University in 2016. The motivation behind the research is to classify and segment 3D representations of images. They use a data structure called point cloud, which is a set of points that represents a 3D shape or object. PointNet provides a unified architecture for applications ranging from object classification, part segmentation, to scene semantic parsing. It is highly efficient and effective, showing strong performance on par or even better than state of the art. \n"
```

Reply:
@OpenVINO-dev-contest can you link to a source of the paper? and mention the researcher name

Reply:
done